ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -6,7 +6,8 @@
     </header>
 
     <div :class="{ 'border-error-left': !getStepValidation(MhrSectVal.LOCATION_VALID)}">
-      <section class="mx-6 pt-8" v-if="!getStepValidation(MhrSectVal.LOCATION_VALID)">
+      <section :class="{ 'pb-8': !(!!getMhrRegistrationLocation.locationType) && !hasAddress }" class="mx-6 pt-8"
+      v-if="!getStepValidation(MhrSectVal.LOCATION_VALID)">
```

Comment:
This condition is a little hard to follow 😁 
Isn't `!(!!getMhrRegistrationLocation.locationType)` the same as `! getMhrRegistrationLocation.locationType` ?
Might be able to simplify that, if so.

Also missing an indent on your v-if. 
Usually, if you require a line break, you would break line for all the props, conditions etc. 
with the `>` on a new line as well, i'm sure there is some examples around.

Reply:
Yes, you're right haha, will change that. Was looking for some examples on the line overlap formatting, will change as you describe